features/stamp_duty_conditional_messaging.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Stamp Duty Information Table
+In order to understand how my stamp duty is calculated
+As a home buyer
+I want to see what percentages apply to the purchase price
+
+Scenario Outline: First time buyer
+  Given I visit the Stamp Duty page
+  And I am a first time buyer
+  When I enter a house price of 490000
+  And I click next
+  Then I should NOT see the first time eligibility message
+
+  Examples:
```

Comment:
Align the stuff so that it's easy to read at first glance?

Reply:
not sure this scenario is correct... think something has changed or been lost (or was never there)
```
Scenario Outline: First time buyer
  Given I visit the Stamp Duty page
  And I am a "<buyer>"
  When I enter a house price of "<price>"
  And I click next
  Then I should NOT see the first time eligibility message
```

Reply:
@willhall88 I've compared this branch with the _master_ and this scenario here is not in the original. I don't know much about this story. What's changed? what's been removed? What's been lost? I don't have anything to compare it with but the knowledge from the _master_ branch.

https://github.com/moneyadviceservice/mortgage_calculator/blob/5e459827424d099fc6e4507cbfce8756f454130a/features/step_definitions/stamp_duty.rb#L114

```
#file: features/step_definitions/stamp_duty.rb

Then(/^I should( NOT)? see the first time eligibility message$/) do |should_not|
    if should_not
      expect(@stamp_duty).to_not have_ftb_conditional
    else
      expect(@stamp_duty).to have_ftb_conditional
    end
end
```